docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DefaultDockerCompose.java

```diff
@@ -194,17 +195,12 @@ public boolean writeLogs(String container, OutputStream output) throws IOExcepti
         try {
             Process executedProcess = logs(container);
             IOUtils.copy(executedProcess.getInputStream(), output);
-            executedProcess.waitFor();
-            return true;
+            return executedProcess.waitFor(LOG_TIMEOUT.getMillis(), TimeUnit.MILLISECONDS);
```

Comment:
We probably want to `destroyForcibly` the process if it times out - I think `waitFor` keeps it running.